<ui version="4.0" >
 <class>MaskExporterDialog</class>
 <widget class="QDialog" name="MaskExporterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Vert</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="check_iom_vertflags" >
          <property name="text" >
           <string>Flags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_vertcolor" >
          <property name="text" >
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_vertquality" >
          <property name="text" >
           <string>Quality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_vertnormal" >
          <property name="text" >
           <string>Normal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_verttexcoord" >
          <property name="text" >
           <string>TextCoord</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Face</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="check_iom_faceflags" >
          <property name="text" >
           <string>Flags</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_facecolor" >
          <property name="text" >
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_facequality" >
          <property name="text" >
           <string>Quality</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_facenormal" >
          <property name="text" >
           <string>Normal</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Wedg</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="check_iom_wedgcolor" >
          <property name="text" >
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_wedgtexcoord" >
          <property name="text" >
           <string>TexCoord</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_iom_wedgnormal" >
          <property name="text" >
           <string>Normal</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Texture Name</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>8</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QListWidget" name="listTextureName" />
        </item>
        <item>
         <widget class="QPushButton" name="renametextureButton" >
          <property name="text" >
           <string>Rename Texture</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QRadioButton" name="AllButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NoneButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_iom_camera" >
       <property name="text" >
        <string>Camera</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" native="1" name="layoutWidget_3" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" native="1" name="layoutWidget" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskExporterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskExporterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
